komga/src/main/kotlin/org/gotson/komga/infrastructure/jooq/ThumbnailSeriesDao.kt

```diff
@@ -66,13 +67,15 @@ class ThumbnailSeriesDao(
     dsl.deleteFrom(ts).where(ts.SERIES_ID.`in`(seriesIds)).execute()
   }
 
-  private fun ThumbnailSeriesRecord.toDomain() =
-    ThumbnailSeries(
-      url = URL(url),
+  private fun ThumbnailSeriesRecord.toDomain(): ThumbnailSeries {
+    return ThumbnailSeries(
+      url = if (!url.isNullOrBlank() && url != "null") URL(url) else null,
```

Comment:
You can't have a blank url, you can simplify using the same form as in the book Dao class